repo.groovy

```diff
@@ -30,6 +30,11 @@ repos = [
     workflowComponent: true,
     chart: 'fluentd'],
 
+  [ name: 'jenkins-node',
```

Comment:
@bacongobbler do you also want the usual component build and release jobs to be auto-generated for `jenkins-node` (really just necessitates that the project have a `Makefile` w/ `docker-build` and `docker-push` targets)?  

If no, we can add an add'l key:value of `buildJobs: false` in this same entry -- then the only job that will be generated will be `jenkins-node-chart-publish` -- which is currently usually kicked off as part of a component's release pipeline (which we wouldn't have w/ `buildJobs: false` mentioned above), but can be kicked off manually as well.